### PR TITLE
Add dns entries into ci scripts

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
Add dns entries into ci scripts

### CODE REVIEW

File: ci/run_premerge_gpu.sh [hunk 1]
```diff
@@ -41,15 +41,29 @@ remove_pipenv() {
     rm Pipfile Pipfile.lock
 }
 
+add_dns_entries() {
+    echo "adding dns entries for HA test cases"
+    cp /etc/hosts temphost
+    echo "localhost0" | tee -a /etc/hosts > /dev/null
```
Comment (pxLi) [inline ci/run_premerge_gpu.sh:47]: `"127.0.0.1 localhost0 localhost1"`

Comment (YuanTingHsieh) [reply ci/run_premerge_gpu.sh]: thanks I fixed it.

**Review by pxLi — APPROVED** [review]
LGTM. lets see if CI could pass as expected

**Pull request merged**